`narn run`

I use `npm run` or `yarn run` often to get a listing of all the scripts.

`narn run` runs `npm run run` which fails.


```
❯ narn run
npm run run
npm ERR! missing script: run
```

--- a/lib/narn-lib.js
+++ b/lib/narn-lib.js
@@ -100,6 +100,8 @@ exports.getNpmArgs = (narnArgs, isPnpm = false) => {
       npmTarget = "run";
       if (narnArgs.length > 1) {
         npmArgs = [narnArgs[0], "--", ...narnArgs.slice(1)];
+      } else if (narnArgs.length === 1 && narnArgs[0] === "run") {
+        npmArgs = [];
       } else {
         npmArgs = narnArgs;
       }

--- a/test/run.test.js
+++ b/test/run.test.js
@@ -1,0 +1,8 @@
+const { getNpmArgs } = require("../lib/narn-lib");
+
+// https://github.com/joeldenning/narn/issues/21
+describe("narn run", () => {
+  it("lists all available commands with npm run", () => {
+    expect(getNpmArgs(["run"])).toEqual(["run"]);
+  });
+});

